frontend/src/utils/helpers/emptyPrintUnauthorized.js

```diff
@@ -0,0 +1,5 @@
+function emptyPrintUnauthorized(param) {
+  return param ? param : 'UNAUTHORIZED_INFO';
```

Comment:
very nit: would it be a better UX to have the Unauth string be more like `******` to represent redacted/restricted info rather than a replacement string of `UNAUTHORIZED_INFO`

Reply:
Done!